apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -119,7 +119,15 @@ const ManagedEventTypeDialog = dynamic(() => import("@components/eventtype/Manag
 
 const AssignmentWarningDialog = dynamic(() => import("@components/eventtype/AssignmentWarningDialog"));
 
-export type Host = { isFixed: boolean; userId: number; priority: number };
+export type Host = {
+  isFixed: boolean;
+  userId: number;
+  priority: number;
+  scheduleId: number | null;
+  availability?: AvailabilityOption | null;
+  avatar?: string;
+  label?: string;
```

Comment:
I don't think we should extend host to include avatar and label especially since we are not storing it. You can use getValue and find avatar and label of the host inside TeamMemberSchedule.

Reply:
This is only for enriching the fetched data, avatar and label are not stored through hosts, these fields are removed before sending payload.
Also wanted to be consistent with Assignment Tab. The label in EventTeamTab (Assignment) is fetched from EventSetupProps.teamMembers. This prop in not available in AvailabilityTab, else requires similar changes.

Reply:
My bad, I meant you can access team members from eventType that is passed as props to UseCommonScheduleSettings by eventType.team.members[number].user and get avatar and label from there. 

Extending hosts is unnecessary as then you would have to clean it up later, best use eventType which is already being passed.